configdefinitions/src/vespa/configserver.def

```diff
@@ -51,9 +51,7 @@ ztsUrl string default=""
 
 # Maintainers
 maintainerIntervalMinutes int default=30
-# TODO: Default set to a high value (1 year) => maintainer will not run, change when maintainer verified out in prod
-tenantsMaintainerIntervalMinutes int default=525600
```

Comment:
Unused?

Reply:
Yes, not in use anymore, it can be removed since it is only used by config server